vm/systemSmartContracts/delegation.go

```diff
@@ -1226,6 +1232,19 @@ func (d *delegation) stakeNodes(args *vmcommon.ContractCallInput) vmcommon.Retur
 	return vmcommon.Ok
 }
 
+func tooManyNodesLogs(logEntries []*vmcommon.LogEntry) bool {
+	for _, logEntry := range logEntries {
+		if len(logEntry.Topics) > 1 {
+			continue
+		}
+		if !bytes.Equal(logEntry.Topics[0], []byte(numberOfNodesTooHigh)) {
```

Comment:
Can't this panic? : `logEntry.Topics[0]`
You check above `if len(logEntry.Topics) > 1`, so this can come down here with `len == 0`

Also, just a question for my own clarification: Is it ok that you check in logs this when calling `stakeNodes` ? Is it 100% sure that this logs will be put here when calling `stakeNodes`?

Reply:
changed to != 1. 

Yes, it is 100% sure, as stakeNodes calls "stake" endpoint from ValidatorSC, which creates this log in those cases.